frontend/tests/config/console.config.yaml

```diff
@@ -0,0 +1,20 @@
+serveFrontend: false
+kafka:
```

Comment:
Should we add the protobuf configuration too, so that protobuf encoded messages will be decoded?